File: ed2/gara4/caffe/att/caffe.pas
function pausa(N: longint; var A, B: array of longint): longint;
begin
    (* Mettete qui il codice della soluzione *)
    pausa := 42;
end;


const
    MAXN = 100000;
var
    N, i  : longint;
    A, B  : array[0..MAXN-1] of longint;
    fr, fw: text;

begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, N);
    for i:=0 to N-1 do
        readln(fr, A[i], B[i]);

    writeln(fw, pausa(N, A, B));
    close(fr);
    close(fw);
end.

File: ed2/gara4/caffe/att/caffe.c
#include <stdio.h>
#include <assert.h>

#define MAXN 100000

int pausa(int N, int A[], int B[]) {
    // Mettete qui il codice della soluzione
    return 42;
}


int A[MAXN], B[MAXN];

int main() {
    FILE *fr, *fw;
    int N, i;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");
    assert(1 == fscanf(fr, "%d", &N));
    for(i=0; i<N; i++)
        assert(2 == fscanf(fr, "%d %d", &A[i], &B[i]));

    fprintf(fw, "%d\n", pausa(N, A, B));
    fclose(fr);
    fclose(fw);
    return 0;
}
